app/src/main/java/com/android/template/ui/RouterImpl.java

```diff
@@ -42,6 +43,11 @@ public boolean showPageInExternalBrowser(final String url) {
         return false;
     }
 
+    @Override
+    public void showWelcomeScreen() {
+        fragmentManager.beginTransaction().replace(CONTAINER_ID, WelcomeFragment.newInstance(), WelcomeFragment.TAG).commit();
```

Comment:
Tvoj je projekt, pa ti nemam kaj prigovarati, ali svakako je preglednije kada wrappaš svaki sljedeći chain-ani poziv:
```
fragmentManager.beginTransaction()
                              .replace(CONTAINER_ID, WelcomeFragment.newInstance()
                              .commit()
                           
```

Reply:
Da, to mi je u biti bio plan već 3 puta napraviti, nikako da se sjetim postaviti opcije